lib/gtl_formatter.rb

```diff
@@ -0,0 +1,107 @@
+class GtlFormatter
+  def self.format_cols(view, row)
+    cols = []
+
+    state = {"state" => :state_format}
+    hardware = {"hardware.bitness" => :hardware_bitness_format}
+    special_cases = {
+      "AutomationRequest"                                              => state,
+      "Container"                                                      => state,
+      "MiqTask"                                                        => state,
+      "MiqProvision"                                                   => state,
+      "MiqRequest"                                                     => state.merge({
+        'approval_state' => :miq_request_format,
+      }),
+      "ManageIQ_Providers_CloudManager_Template-all_vms_and_templates" => hardware,
+      "ManageIQ_Providers_CloudManager_Vm-vms"                         => hardware,
+      "ManageIQ_Providers_CloudManager_Vm-all_vms_and_templates"       => hardware,
+      "ManageIQ_Providers_CloudManager_Vm"                             => hardware,
+      "ManageIQ_Providers_CloudManager_Template"                       => hardware.merge({
+        "image?" => :cloud_manager_template_format,
+      }),
+      "MiqSchedule" => :timezone, # all fields have same specific format
```

Comment:
Looks like the only fields that really need this are `last_run_on` and `next_run_on`.
So, maybe we don't need an "all cols" case after all.
(Not sure if that's a change for this PR?)

Reply:
Let's leave it to followup PR :)